app/javascript/mastodon/actions/compose.js

```diff
@@ -244,6 +248,13 @@ export function unmountCompose() {
   };
 };
 
+export function changeComposeAdvancedOption(option) {
```

Comment:
Not a problem at all, but it might be clearer to call this `toggleComposeAdvancedOption` since you don't provide the option of specifying what to change it *to*